README.md

```diff
@@ -136,6 +186,11 @@ use code like this.
 
 ```
 
+```Yammer Metrics 2.0.x
+
+NOT SUPPORTED;
```

Comment:
There is not method MetricsRegistry::register(...)
So we can not offer user helper functionality to track duplicates in created Metrics instances